.github/scripts/python.sh

```diff
@@ -34,7 +34,7 @@ cd $GITHUB_WORKSPACE
 cd $GITHUB_WORKSPACE
 
 # check that main script executes on toy Door dataset
-python gtsfm/runner/run_scene_optimizer.py
+python gtsfm/runner/run_scene_optimizer.py --config_name sift_front_end.yaml
```

Comment:
CI took an hour with the default front-end (deep_front_end.yaml), so switching to one that takes 15 minutes to not burn up our CI quota.

Reply:
so this 1-line change I had to make as a hotfix to the Github Actions CI file here in master, to prevent us from running out of CI $$. So it's in master already now.